lib/schedule.explorer.ts

```diff
@@ -26,19 +26,20 @@ export class ScheduleExplorer implements OnModuleInit {
       ...this.discoveryService.getControllers(),
       ...this.discoveryService.getProviders(),
     ];
-    instanceWrappers
-      .filter((wrapper) => wrapper.isDependencyTreeStatic())
-      .forEach((wrapper: InstanceWrapper) => {
-        const { instance } = wrapper;
-        if (!instance || !Object.getPrototypeOf(instance)) {
-          return;
-        }
-        this.metadataScanner.scanFromPrototype(
-          instance,
-          Object.getPrototypeOf(instance),
-          (key: string) => this.lookupSchedulers(instance, key),
-        );
-      });
+    instanceWrappers.forEach((wrapper: InstanceWrapper) => {
+      const { instance } = wrapper;
```

Comment:
I'm not exactly sure the implication of getting an instance for a request scoped provider here. It seems like it uses a static context id when resolving the instance, but I do not know if this could cause any issues.

Reply:
This should be fine! For request-scoped providers, `instance` would be a prototype object